src/routes/(app)/admin[[hash=admin_hash]]/reporting/+page.svelte

```diff
@@ -444,59 +495,54 @@
 						<th class="border border-gray-300 px-4 py-2">Order ID</th>
 						<th class="border border-gray-300 px-4 py-2">Order Date</th>
 						<th class="border border-gray-300 py-2">Currency</th>
-						<th class="border border-gray-300 px-4 py-2">Amount</th>
+						<th class="border border-gray-300 px-4 py-2">Price</th>
 						<th class="border border-gray-300 px-4 py-2">Vat Rate</th>
 					</tr>
 				</thead>
 				<tbody>
 					<!-- Order rows -->
 					{#each orderFiltered as order}
 						{#each order.items as item}
-							<tr class="hover:bg-gray-100 whitespace-nowrap">
-								<td class="border border-gray-300 px-4 py-2"
-									>{data.websiteLink + '/product/' + item.product._id}</td
-								>
-								<td class="border border-gray-300 px-4 py-2">{item.product.name}</td>
-								<td class="border border-gray-300 px-4 py-2">{item.quantity}</td>
-								<td class="border border-gray-300 px-4 py-2">{item.depositPercentage ?? 100}</td>
-								<td class="border border-gray-300 px-4 py-2">{order.number}</td><td
-									class="border border-gray-300 px-4 py-2"
-								>
-									<time
-										datetime={order.createdAt.toISOString()}
-										title={order.createdAt.toLocaleString($locale)}
+							{#if !data.tagId || item.product.tagIds?.includes(data.tagId)}
+								<tr class="hover:bg-gray-100 whitespace-nowrap">
+									<td class="border border-gray-300 px-4 py-2"
+										>{data.websiteLink + '/product/' + item.product._id}</td
 									>
-										{order.createdAt.toLocaleDateString($locale)}
-									</time>
-								</td>
-								<td class="border border-gray-300 px-4 py-2">{data.currencies.main}</td>
-								<td class="border border-gray-300 px-4 py-2"
-									>{(toCurrency(
-										data.currencies.main,
-										(item.customPrice?.amount ?? item.product.price.amount) *
-											(item.discountPercentage ? (100 - item.discountPercentage) / 100 : 1),
-										item.customPrice?.currency ?? item.product.price.currency
-									) *
-										(item.product.deposit?.percentage ?? 100) *
```

Comment:
Note @Tirodem : I removed the prorata of product price * deposit in "product details" - it doesn't take into account if the product was actually paid with a deposit or not, or even if it was, if the rest of the order was paid later on.

it's very easy to add back (just add back this line)